tools/experiments.md

```diff
@@ -17,7 +17,7 @@ The work is being tracked in
 * Further reading:
     * https://wiki.mozilla.org/Firefox/SHIELD
     * https://wiki.mozilla.org/Firefox/Shield/Shield_Studies
-    * https://mozilla.github.io/shield-studies-docs/study-process/
+    * [https://mozilla.github.io/shield-studies-docs/study-process/](BROKEN:https://mozilla.github.io/shield-studies-docs/study-process/)
```

Comment:
Note: I don't know what the real links for this (and the other one in this file) are supposed to be.